index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap');
+      </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main" class="container">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div>The man who saved Billions</div>
+
+      <div id="img-div">
+ 
+      <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="A Picture of Dr. Norman and colleagues" id="image">
+      <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+      </div>
+
+    <div id="tribute-info">
```

Comment:
HTML and CSS best practices:
- Try using `section` instead of `div`

Reply:
Great working on the inlince comments but I think here it would be great if you explain the reason why you are asking since you are saying it is just confusing but you didn't mention the reason because may be it is obvious to you but not to others

> You can say 
I Think it would be better if you use `section` tag instaed of `div` tag 🚀 because it is important to make you HTML page semantic and follow the best practice 🚀✅✅🔥🔥